openmdao/core/driver.py

```diff
@@ -48,6 +48,8 @@ def __init__(self):
         self.root = None
 
         self.iter_count = 0
+        self.dv_conversions = {}
```

Comment:
What do you mean? They are already initialized in _setup.  They are just declared in the constructor.